### PR TITLE
Update wkw variables description

### PR DESCRIPTION
**Pull request by mbenguig**

### CODE REVIEW

File: JobExecTimeWorkflows/resources/catalog/variables_propagation_48hours.xml [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
-        <variable name="coveredAnglePerLight" value="90" model="PA:INTEGER"/>
-        <variable name="currentCoverage" value="0" model="PA:INTEGER"/>
+        <variable name="numberOfLights" value="1" model="PA:LIST(1,2,3,4,5)" description="The number of lights."/>
+        <variable name="minRequiredAngle" value="120" model="PA:INTEGER" description="The min angle in degrees to cover."/>
+        <variable name="coveredAnglePerLight" value="90" model="PA:INTEGER" description="The angle in degrees covered by a single light."/>
+        <variable name="currentCoverage" value="0" model="PA:INTEGER" description="The angle in degrees currently covered by lights."/>
     </variables>
     <description>
-        <![CDATA[ Demonstrate how variables can be read/written to in different script tasks. ]]>
+        <![CDATA[ Demonstrate how variables can be read/written to in different script tasks. In this use case, we looks for the minimum number of lights to cover a minimum angle. Its priority is adapted to target a 48h execution time. ]]>
```
Comment (aminelouati) [inline JobExecTimeWorkflows/resources/catalog/variables_propagation_48hours.xml:12]: we looks -> we look

Comment (Hiba-Alili) [inline JobExecTimeWorkflows/resources/catalog/variables_propagation_48hours.xml:12]: Same

File: JobExecTimeWorkflows/resources/catalog/variables_propagation_2hours.xml [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
-    <variable name="coveredAnglePerLight" value="90" model="PA:INTEGER"/>
-    <variable name="currentCoverage" value="0" model="PA:INTEGER"/>
+    <variable name="numberOfLights" value="1" model="PA:LIST(1,2,3,4,5)" description="The number of lights."/>
+    <variable name="minRequiredAngle" value="120" model="PA:INTEGER" description="The min angle in degrees to cover."/>
+    <variable name="coveredAnglePerLight" value="90" model="PA:INTEGER" description="The angle in degrees covered by a single light."/>
+    <variable name="currentCoverage" value="0" model="PA:INTEGER" description="The angle in degrees currently covered by lights."/>
   </variables>
   <description>
-    <![CDATA[ Demonstrate how variables can be read/written to in different script tasks. ]]>
+        <![CDATA[ Demonstrate how variables can be read/written to in different script tasks. In this use case, we looks for the minimum number of lights to cover a minimum angle. Its priority is adapted to target a 2h execution time. ]]>
```
Comment (Hiba-Alili) [inline JobExecTimeWorkflows/resources/catalog/variables_propagation_2hours.xml:12]: The same

File: JobExecTimeWorkflows/resources/catalog/variables_propagation_1minute.xml [hunk 3]
```diff
@@ -3,13 +3,13 @@
...
-        <variable name="coveredAnglePerLight" value="90" model="PA:INTEGER"/>
-        <variable name="currentCoverage" value="0" model="PA:INTEGER"/>
+        <variable name="numberOfLights" value="1" model="PA:LIST(1,2,3,4,5)" description="The number of lights."/>
+        <variable name="minRequiredAngle" value="120" model="PA:INTEGER" description="The min angle in degrees to cover."/>
+        <variable name="coveredAnglePerLight" value="90" model="PA:INTEGER" description="The angle in degrees covered by a single light."/>
+        <variable name="currentCoverage" value="0" model="PA:INTEGER" description="The angle in degrees currently covered by lights."/>
     </variables>
     <description>
-        <![CDATA[ Demonstrate how variables can be read/written to in different script tasks. ]]>
+        <![CDATA[ Demonstrate how variables can be read/written to in different script tasks. In this use case, we looks for the minimum number of lights to cover a minimum angle. Its priority is adapted to target a 1min execution time. ]]>
```
Comment (Hiba-Alili) [inline JobExecTimeWorkflows/resources/catalog/variables_propagation_1minute.xml:12]: Same here

**Review by Hiba-Alili — APPROVED** [review]

**Pull request merged**